Directory.Packages.props

```diff
@@ -23,9 +23,9 @@
     <PackageVersion Include="Moq" Version="4.18.2" />
     <PackageVersion Include="xunit" Version="2.4.2" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
-    <PackageVersion Include="Serilog" Version="2.12.0" />
+    <PackageVersion Include="Serilog" Version="3.1.1" />
```

Comment:
Are there any breaking changes in Serilog 3 that would be of concern pushing this out as a minor release?

Reply:
I haven't checked yet, but good point.